Library/Homebrew/extend/os/linux/dependency_collector.rb

```diff
@@ -0,0 +1,5 @@
+class DependencyCollector
+  def ant_dep(spec, tags)
+    Dependency.new(spec.to_s, tags)
+  end
```

Comment:
Like what we're going for is the code in `os` being OS-specific (including macOS specific) whereas this version is generic enough to be the default.